Pod/Classes/Highlightr.swift

```diff
@@ -165,7 +166,18 @@ open class Highlightr
     }
     
     //Private & Internal
-    fileprivate func processHTMLString(_ string: String) -> NSAttributedString?
+    /**
+     Execute block safely on main thread, execute block directly if current thread is main thread, otherwise execute block on main queue synchronously.
+     */
+    private func safeMainSync(_ block: @escaping ()->()) {
+        if Thread.isMainThread {
+            block()
+        } else {
```

Comment:
Please follow [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) for indentation.